### PR TITLE
Support for device model name property and has capability method

### PR DESCRIPTION
**Pull request by jsetton**
Add the ability to get a given device model name and determine its capabilities.
Fixes: #112 
Fixes: #114

### CODE REVIEW

File: ring_doorbell/doorbot.py [hunk 1]
```diff
@@ -29,6 +29,24 @@ def family(self):
         """Return Ring device family type."""
         return 'doorbots'
 
+    @property
+    def model(self):
+        """Return Ring device model name."""
+        if self.kind in ['doorbot', 'doorbell', 'doorbell_v3']:
```
Comment (tchellomello) [inline ring_doorbell/doorbot.py]: How about if we make the kinds a CONSTANT so then it would be easier to maintain the code? What do you think?

Comment (jsetton) [reply ring_doorbell/doorbot.py]: What do you think about using this structure?

```
# device model kinds
CHIME_KINDS = ['chime']
CHIME_PRO_KINDS = ['chime_pro']

DOORBELL_KINDS = ['doorbot', 'doorbell', 'doorbell_v3']
DOORBELL_2_KINDS = ['doorbell_v4', 'doorbell_v5']
DOORBELL_PRO_KINDS = ['lpd_v1', 'lpd_v2']
DOORBELL_ELITE_KINDS = ['jbox_v1']

FLOODLIGHT_CAM_KINDS = ['hp_cam_v1']
SPOTLIGHT_CAM_BATTERY_KINDS = ['stickup_cam_v4']
SPOTLIGHT_CAM_WIRED_KINDS = ['hp_cam_v2']
STICKUP_CAM_KINDS = ['stickup_cam', 'stickup_cam_v3']
STICKUP_CAM_BATTERY_KINDS = ['stickup_cam_lunar']
STICKUP_CAM_WIRED_KINDS = ['stickup_cam_elite']

# device wired kinds
DOORBOT_WIRED_KINDS = (
    DOORBELL_PRO_KINDS + 
    DOORBELL_ELITE_KINDS
)
STICKUP_CAM_WIRED_KINDS = (
    FLOODLIGHT_CAM_KINDS +
    SPOTLIGHT_CAM_WIRED_KINDS + 
    STICKUP_CAM_WIRED_KINDS
)

```

Comment (tchellomello) [reply ring_doorbell/doorbot.py]: +1 that looks great!!! It will be so much easier to maintain later on!! Good job!

**Comment by jsetton** [comment]
I made the change as agreed. In the process, I swapped the wired property implementation for a capability check method, which is more in line with the capabilities matrix I compiled in my original issue.

The following capability can be determined for each device:

- battery
- light
- siren
- volume

**Comment by jsetton** [comment]
@tchellomello are you fine with the changes I made?

**Pull request merged**